themes/bootstrap3/js/combined-search.js

```diff
@@ -0,0 +1,22 @@
+/*global VuFind, setupOpenUrlLinks, checkSaveStatuses, setupSaveRecordLinks */
+VuFind.combinedSearch = (function() {
```

Comment:
(VuFind) removed. There shouldn't be anything to worry, all this should work fine with the existing code, but of course I might have messed up something..